starter-code/index.js

```diff
@@ -1,17 +1,67 @@
 class SortedList {
-  constructor() {}
+  constructor(items,length) {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.length++;
+    this.items.sort(function(a,b){
```

Comment:
El método sort, si no le pasas nada, ya hace un a - b si no recuerdo mal. `this.items.sort()`